utils/plugin_docs.py

```diff
@@ -0,0 +1,59 @@
+
+import os
+from distutils.dir_util import copy_tree
+from jinja2 import Environment, FileSystemLoader
+
+
+# Directory in plugin to pull documentation from
+PLUGIN_DOCS_DIR = 'docs'
+
+
+def import_plugin_docs(caldera_root_dir, sphinx_root_dir):
+    print(f'Importing plugin documentation')
```

Comment:
Let's either remove this or change it to an `info` log message. My rule is never `print` anything from a function unless that is the purpose of the function. If you want to display information/debug calls from a utility function, use logging instead so that people can shut it off or configure thresholds if they want.